rules/cattle-flu.smk

```diff
@@ -96,3 +96,31 @@ rule prune_tree:
             --output-tree {output.tree} \
             --output-metadata {output.node_data}
         """
+
+def assert_expected_config(w):
+    try:
+        assert len(config['traits']['genome_columns'])==1 and config['traits']['genome_columns']['FALLBACK']=="division"
+    except Exception as err:
+        print(str(err))
+        raise Exception("Rule add_metadata_columns_to_show_non_inferred_values expected a certain format for config['traits'] that has since changed") from err
+
+rule add_metadata_columns_to_show_non_inferred_values:
+    """
+    Genome builds run `augur traits` for "division" (we assert this below) so we want to add a metadata
+    column `division_metadata` which is a duplicate of `division`.
+
+    NOTE: long-term we should be consulting `traits_params()` to work out the columns to duplicate, but
+    that function's not visible to this .smk file so would require deeper refactoring.
```

Comment:
Nice! I was thinking we'd have to pull them out into a common file. Would this have any side-effects? E.g. if a main-snakefile rule targeted a filename from the custom rules file is this ok? What about if it referenced `rule.x.output.y` instead of the filename?

Reply:
Using the `include` directive at the end would be equivalent to just including all of contents of `cattle-flu.smk` at the end of the file. Access to target filenames and `rules.x.output.y` should still work. Just any variables and/or functions defined in `cattle-flu.smk` would not be accessible in the main Snakefile.

Reply:
(P.S. once we do this, I have some WIP commits that shift more of the cattle-flu stuff into `cattle-flu.smk`, but which can't currently do so because they need access to functions defined in the main snakefile)

Reply:
(Leaving this for future work)